website/docs/prysm-usage/slasher.md

```diff
@@ -3,71 +3,41 @@ id: slasher
 title: Running a slasher
 sidebar_label: Running a slasher
 ---
-This section provides instructions to run [slasher](https://github.com/prysmaticlabs/prysm/tree/master/slasher) to identify and report slashable offenses on eth2 .  If slasher detects a slashable offense, proof is sent to the [beacon-chain node](https://docs.prylabs.network/docs/how-prysm-works/beacon-node/) for inclusion in a block.  [Validators](https://docs.prylabs.network/docs/how-prysm-works/prysm-validator-client/) earn a small whistleblower reward for including this proof into a block.  
+This section provides instructions to run [slasher](https://github.com/prysmaticlabs/prysm/tree/master/slasher) as an **optional** process to report slashable offenses on eth2 .  If slasher detects a slashable offense, proof is sent to the [beacon-chain node](https://docs.prylabs.network/docs/how-prysm-works/beacon-node/) for inclusion in a block.  [Validators](https://docs.prylabs.network/docs/how-prysm-works/prysm-validator-client/) earn a small whistleblower reward for including this proof into a block.  
 
-:::danger Slasher Can Be a Resource Hog
-Slasher can be a huge resource hog during times of no chain finality, which can manifest as massive RAM usage. Please make sure you understand the risks of this, especially if you want high uptime for your beacon nodes. Slasher places significant stress on beacon nodes when the chain has no finality, and might be the reason why your validators are underperforming if your beacon node is under this much stress.
+:::tip Slasher Requires Significant Disk Space
+Slasher is essentially a beacon node with **superpowers**. It might use over 70Gb of disk space mainnet today, and we recommend running a slasher with more than the minimal specifications in our [installation guides](/docs/install/install-with-script) and instead running the **recommended** specifications.
 :::
 
-> NOTICE: Running slasher is optional, but helps secure the chain and may result in additional earnings.
+## What is Slashing
 
-## Slashable Offenses
+Slashing occurs when a validator has provably acted against the Ethereum network. Ethereum proof of stake works on a penalty-based incentive mechanism to heavily discourage actions on the network that could cause instability, malicious forking, and conflicting information from validators. Slashing doesn’t need to have malicious intent necessarily, however, as it could happen from misconfiguration as well. If a validator acts in a way that can confuse or disrupt the integrity of the system, the protocol removes, or **slashes**, a portion of the offending validators existing stake, causing a gradual loss of ETH over time until the validator is forcefully ejected and marked as SLASHED. This is **irreversible**.
```

Comment:
```suggestion
Slashing occurs when a validator has provably acted against the Ethereum network. Ethereum proof of stake works on a penalty-based incentive mechanism to heavily discourage actions on the network that could cause instability, malicious forking, and conflicting information from validators. Slashing doesn’t need to have malicious intent necessarily, however, as it could happen from misconfiguration as well. If a validator acts in a way that can confuse or disrupt the integrity of the system, the protocol removes, or **slashes**, a portion of the offending validator's existing stake, causing a gradual loss of ETH over time until the validator is forcefully ejected and marked as SLASHED. This is **irreversible**.
```